wso2is_analytics/README.md

```diff
@@ -16,13 +16,18 @@ environments. It supports multiple versions of WSO2 Data Analytics Server. Confi
 
 * Setup the puppet environment with the puppet modules wso2is_analytics and wso2base.
 * WSO2 IS Analytics 5.4.0  puppet modules are compatible and tested with
-[puppet-base](https://github.com/wso2/puppet-base/) version 1.0.0 and [puppet-common](https://github.com/wso2/puppet-common) version 1.0.0
-* So if using puppet-common's setup.sh to setup the PUPPET_HOME, use this version (1.0.0) of puppet-common.
+[puppet-base](https://github.com/wso2/puppet-base/) version 1.2.0 and [puppet-common](https://github.com/wso2/puppet-common) version 1.1.0
+* So if using puppet-common's setup.sh to setup the PUPPET_HOME, use this version (1.1.0) of puppet-common.
 * After setting up PUPPET_HOME using puppet-common's setup.sh, checkout the above mentioned compatible version of puppet-base.
 
 ## Setup a Development Environment
 Follow the steps mentioned in the [wiki](https://github.com/wso2/puppet-base/wiki) to setup a development environment and update/create new puppet modules.
 
+## Dependencies
```

Comment:
Not needed because this is already covered by "Let's tell "WSO2 IS Analytics 5.4.0 puppet module is compatible and tested with .....".